experimental/LieAlgebras/src/LieAlgebras.jl

```diff
@@ -83,8 +83,9 @@ import ..Oscar:
   ⊕,
   ⊗
 
-import Base: getindex, deepcopy_internal, hash, issubset, iszero, parent, zero
+import Base: getindex, deepcopy_internal, hash, issubset, iszero, max, parent, zero
 
+export AbstractCrystal, AbstractCrystalElem
```

Comment:
off-topic question for @lgoettgens: these `export` statements all have to be duplicate. Would it make sense to put them into an `exports.jl` and then just `include` that twice, in both locations?

Reply:
That's a great idea. I just checked, and the two lists (already on current master), differ by some things (some names get exported just into the Oscar namespace but not exported from Oscar, and thus make it easier for other devs to use them and thus work against https://github.com/oscar-system/Oscar.jl/issues/3408 by accident). I'll prepare a PR for streamlining the two lists and putting them somewhere separate. This PR can then get rebased afterwards.